.github/CODEOWNERS

```
@@ -1 +1,2 @@
-* mohit@deepsource.io
\ No newline at end of file
+* mohit@deepsource.io
+* jai@deepsource.io
```

```suggestion
* jai@deepsource.io mohit@deepsource.io
```

If we have it in two lines, both code owners need to approve before the PR can be merged. Not sure if that was intentional. With the suggestion, only one approval is necessary.